shared/getDeviceOnDash/artik10.md

```diff
@@ -1,22 +1,22 @@
-## Setting Up Your Device
+In order to get **resinOS** up and running on your {{ $device.name }}, you need to first set it up to boot from your {{ $device.bootMedia }} rather than its internal [eMMC memory][emmc-link]. To do this, you need to set the tiny `SW2` dip switch to position `1:on` and `2:on`, as shown below.
 
-In order to get the resinOS up and running on our {{ $device.name }} we need to first set it up to boot from our {{ $device.bootMedia }} rather than its internal [eMMC memory][emmc-link]. To do this we need to set the tiny `SW2` dip switch to position `1:on` and `2:on`, as shown below.
-
-__Note:__ This resinOS will completely write over the existing eMMC.
+__Note:__ **ResinOS** will completely write over the existing eMMC.
 
 <img src="/img/artik10/artik10-dev-kit.png" width="80%">
 
-We can now insert the 5VDC power cable and flip the power, switch labelled `PSW1`, to the `on` position. We should now have some glowing LEDs indicating a sign of life. Next we need to press and hold the `SW3 POWER` push button for 1 or 2 seconds, this starts the boot from the {{ $device.bootMedia }}.
+We can now insert the 5VDC power cable and flip the power, switch labelled `PSW1`, to the `on` position. You should now have some glowing LEDs indicating a sign of life. Next, you need to press and hold the `SW3 POWER` push button for 1 or 2 seconds, this starts the boot from the {{ $device.bootMedia }}.
```

Comment:
For consistency, I guess the first "We" on this line should be changed to "You" too.